spec/features/activity_page_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
-describe 'activity page', type: :feature, js: true do
+describe 'activity page', type: :feature, js: true, feature: true do
```

Comment:
Yep. I just didn't look far enough to find it. New version does that: `--tag type:feature`.  If we want to break up those feature specs into a bunch of different categories for even more builds, we may need to add some tags though.